android/assets/jsons/translations/Simplified_Chinese.properties

```diff
@@ -3656,7 +3562,7 @@ Heal adjacent units for an additional 15 HP per turn = 相邻的所有单位每
 
 
  # Requires translation!
-Why do we fight' Because Inanna demands it. Now, witness the power of the Sumerians! = 
+Why do we fight' Because Inanna demands it. Now, witness the power of the Sumerians! = 为什么我们要战斗？因为伊娜娜要求。现在，看看苏美尔人的力量！
```

Comment:
最好翻译成伊南娜女神，她好像是苏美尔人的战斗女神，要求改成召唤

Reply:
@yairm210 this sentence "Why do we fight' Because Inanna demands it. Now, witness the power of the Sumerians!" has been replace with "Why do we fight? Because Inanna demands it. Now, witness the power of the Sumerians!" and translated in #2804, so it is very odd.
The city name translated by the contributor is very perfect.